addons/l10n_id/models/res_bank.py

```diff
@@ -91,6 +91,9 @@ def _get_qr_vals(self, qr_method, amount, currency, debtor_partner, free_communi
                 "cliTrxAmount": int(amount)
             }
             response = _l10n_id_make_qris_request('show_qris.php', params)
+            if response.get("status") == "failed":
+                msg = response.get("data")
+                raise ValidationError(_("%s", msg))
```

Comment:
There shouldn't be any need to translate this (unless we use a prefix or something like @nni-odoo 's suggestion)

Reply:
Yes, you are correct
I totally miss this 😄 

Thanks :)